### PR TITLE
Feature/Execution Service

### PR DESCRIPTION
**Pull request by pgorgolew**

### CODE REVIEW

**Review by proman3419 — CHANGES_REQUESTED** [review]

File: executor/src/main/java/org/k8loud/executor/controller/ExecutorController.java [hunk 1]
```diff
@@ -9,16 +9,14 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-public class ExecutionController {
-
-
+public class ExecutorController {
```
[the remark this answers is not in the document]
Comment (pgorgolew) [reply executor/src/main/java/org/k8loud/executor/controller/ExecutorController.java:12]: I will change

File: executor/src/main/java/org/k8loud/executor/service/ExecutionWrapperServiceImpl.java [hunk 2]
```diff
@@ -24,10 +25,9 @@ public ResponseEntity<String> execute(ExecutionRQ request) {
         //TODO interceptor
         Action action = mapperService.map(request);
         if (action == null) {
-            executionService.execute(action);
-        } else {
             return new ResponseEntity<>("dupa", HttpStatus.BAD_REQUEST);
         }
-        return new ResponseEntity<>("Jest git", HttpStatus.OK);
+        ExecutionRS response = executionService.execute(action);
+        return new ResponseEntity<>("Action response: " + response, HttpStatus.OK);
```
Comment (proman3419) [inline executor/src/main/java/org/k8loud/executor/service/ExecutionWrapperServiceImpl.java:31]: Is the "Action response: " needed?

File: executor/src/main/java/org/k8loud/executor/service/ExecutionServiceImpl.java [hunk 3]
```diff
@@ -9,7 +10,8 @@
 public class ExecutionServiceImpl implements ExecutionService {
 
     @Override
-    public void execute(Action action) {
-
+    public ExecutionRS execute(Action action) {
```
Comment (proman3419) [inline executor/src/main/java/org/k8loud/executor/service/ExecutionServiceImpl.java:13]:
```
@NotNull
ExecutionRS execute(@NotNull Action action);
```

File: executor/src/main/java/org/k8loud/executor/service/ExecutionService.java [hunk 4]
```diff
@@ -1,8 +1,9 @@
 package org.k8loud.executor.service;
 
+import data.ExecutionRS;
 import org.k8loud.executor.action.Action;
 
 public interface ExecutionService {
 
-    void execute(Action action);
+    ExecutionRS execute(Action action);
```
Comment (proman3419) [inline executor/src/main/java/org/k8loud/executor/service/ExecutionService.java:8]:
```
@NotNull
ExecutionRS execute(@NotNull Action action);
```

**Review by proman3419 — CHANGES_REQUESTED** [review]

File: executor/src/main/java/org/k8loud/executor/service/ExecutionWrapperServiceImpl.java (hunk 2 above)
Comment (pgorgolew) [reply executor/src/main/java/org/k8loud/executor/service/ExecutionWrapperServiceImpl.java:31]: I feeling like we should parser this response and create some kind of responses

**Review by proman3419 — APPROVED** [review]

**Pull request merged**